### PR TITLE
Committee導入

### PR DESCRIPTION
**Pull request by atsuya02**
## 概要
テストで[committee-rails](https://github.com/willnet/committee-rails)使用

### CODE REVIEW

File: server/openapi.yml [hunk 1]
```diff
@@ -461,8 +459,6 @@ components:
   responses:
     NoContent:
       description: リクエストが成功し、返すリソースがない時のレスポンス
-      content:
-        {}
```
Comment (atsuya02) [inline server/openapi.yml:465]: > Empty Response Body
Some responses, such as 204 No Content, have no body. To indicate the response body is empty, do not specify a content for the response:

https://swagger.io/docs/specification/describing-responses/

File: server/openapi.yml [hunk 2]
```diff
@@ -522,6 +518,7 @@ components:
       type: object
       required:
         - messages
+      additionalProperties: false
```
Comment (atsuya02) [inline server/openapi.yml:521]: swaggerで定義していない属性をレスポンスボディで返すとテストが落ちるようになる
https://zenn.dev/leaner_tech/articles/20210813-openapi-additional-properties

https://swagger.io/specification/?sbsearch=additionalProperties
> additionalProperties - Value can be boolean or object. Inline or referenced schema MUST be of a [Schema Object](https://swagger.io/specification/?sbsearch=additional-properties#schema-object) and not a standard JSON Schema. Consistent with JSON Schema, additionalProperties defaults to true.

**Comment by atsuya02** [comment]
@YuyaYoshioka 
レビューお願い

**Review by YuyaYoshioka — APPROVED** [review]
LGTM

additionalPropertiesのdefaultを一括でfalseにしたかったけど，調べた感じいい方法がなさそうだった。falseの書き忘れが嫌やな。

**Pull request merged**